site/components/RadioExample.tsx

```diff
@@ -0,0 +1,48 @@
+import { css } from 'emotion';
+import { h } from 'preact';
+import { RoutableProps } from 'preact-router';
+import { useState } from 'preact/hooks';
+
+import { Radio } from '../../src/Radio';
+
+enum Answers {
```

Comment:
Do we really need to show the used by means of Enum? It is entirely optional. We can rely on simple string usage?